src/Game/UI/Gumps/PaperdollGump.cs

```diff
@@ -433,17 +433,24 @@ private void PartyManifest_MouseDoubleClickEvent(object sender, MouseDoubleClick
         {
             if (args.Button == MouseButtonType.Left)
             {
```

Comment:
This one is questionable for upstream, but ClassicUO already does have CUOEnvironment.IsOutlands and displays the custom status bar, so I figured why not.